pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["setuptools"]
```

Comment:
You should add a lower bound of `setuptools >= 61.0.0` as that's when setuptools added support for pyproject.toml. If pip or other build tools end up using an earlier version it will fail to build your package.